### DISCUSSION TITLE
React 18 compatibility

### DISCUSSION THREAD

**Discussion by itakiyev** [microsoft/AdaptiveCards#8671] [Q&A]
Good day!

May I know if there are any plans to continue working on [adaptivecards-react](https://www.npmjs.com/package/adaptivecards-react) library so that is is supported in latest React JS versions?

currently it doesnt seem to be supported on ^18.2.0 react version

regards,

**Comment by jwoo-msft** [comment]
yes, we don't have a plan to actively support it.

**Comment by varatharajb** [comment]
Hello [jwoo-msft](https://github.com/jwoo-msft)

 Is there any plan extend the adaptive card support for Reat18?
I am asking this question because  your last comment was made almost an year ago.
